crypten/mpc/mpc.py

```diff
@@ -769,9 +787,15 @@ def _max_pool2d_backward(
             stride = kernel_size
         if isinstance(stride, int):
             stride = stride, stride
-        assert isinstance(padding, tuple), "stride must be a int, tuple, or None"
+        assert isinstance(stride, tuple), "stride must be a int, tuple, or None"
         s0, s1 = stride
 
+        # Setup dilation
+        if isinstance(stride, int):
```

Comment:
BUG: Should be ```dilation``` and at the assert it should be
```"dilation must be a int or tuple``` (the default value is set to 1)